src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -376,4 +369,27 @@ class EthService(
     }
   }
 
+  private def doCall(req: CallRequest): Either[JsonRpcError, Ledger.TxResult] = {
+    val fromAddress = req.tx.from
+      .map(Address.apply) // `from` param, if specified
+      .getOrElse(
+      keyStore
+        .listAccounts().getOrElse(Nil).headOption // first account, if exists and `from` param not specified
+        .getOrElse(Address(0))) // 0x0 default
+
+    val toAddress = req.tx.to.map(Address.apply)
+
+    val gasLimit = req.tx.gas.getOrElse {
+      resolveBlock(BlockParam.Latest) match {
+        case Right(b) => b.header.gasLimit
+        case Left(err) => BigInt(0)
```

Comment:
if latest is not avaliable maybe we can provide some constant value that allows to execute transaction, with 0 tx will always fail, it requires gas even to do ether transfer
we can also treat case when we are not able to get latest block as error and return JsonRpcError for internal error